src/components/ec-file-list/ec-file-list.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    data-test="ec-file-list"
+    class="ec-file-list"
+  >
+    <div
+      v-for="(item, index) in items"
+      :key="item.name"
+      ref="items"
+      class="ec-file-list__item"
+      :data-test="`ec-file-list__item ec-file-list__item--${index}`"
+    >
+
+      <ec-icon
```

Comment:
I think we need to be able to display loading indicator here as well. I'm not sure if that's part of the ticket.

Reply:
Good remembering, still needs to be discussed so not part of this ticket.

Reply:
indeed we said to do the loading in another story